From 9cff8dd1c0bf1ef07595fee080dcef3978a0b991 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Mon, 10 Dec 2012 15:38:28 +0100
Subject: [PATCH] build: link omelasticsearch against -lm

Use LT_LIB_M to find the math library which is needed for pow().
---
 configure.ac                        |    2 ++
 plugins/omelasticsearch/Makefile.am |    2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 1b203ac..0088672 100644
--- a/configure.ac
+++ b/configure.ac
@@ -720,6 +720,8 @@ AC_ARG_ENABLE(elasticsearch,
 )
 if test "x$enable_elasticsearch" = "xyes"; then
 	PKG_CHECK_MODULES([CURL], [libcurl])
+	LT_LIB_M
+
 fi
 AM_CONDITIONAL(ENABLE_ELASTICSEARCH, test x$enable_elasticsearch = xyes)
 
diff --git a/plugins/omelasticsearch/Makefile.am b/plugins/omelasticsearch/Makefile.am
index 059bdf8..ba85a89 100644
--- a/plugins/omelasticsearch/Makefile.am
+++ b/plugins/omelasticsearch/Makefile.am
@@ -4,6 +4,6 @@ pkglib_LTLIBRARIES = omelasticsearch.la
 omelasticsearch_la_SOURCES = omelasticsearch.c cJSON/cjson.c  cJSON/cjson.h
 omelasticsearch_la_CPPFLAGS =  $(RSRT_CFLAGS) $(PTHREADS_CFLAGS)
 omelasticsearch_la_LDFLAGS = -module -avoid-version
-omelasticsearch_la_LIBADD =  $(CURL_LIBS)
+omelasticsearch_la_LIBADD =  $(CURL_LIBS) $(LIBM)
 
 EXTRA_DIST = 
-- 
1.7.10.4

